src/main/scala/io/keen/client/scala/EventStore.scala

```diff
@@ -0,0 +1,21 @@
+package io.keen.client.scala
+
+import scala.collection.concurrent.TrieMap
+import scala.collection.mutable.ListBuffer
+
+trait EventStore {
+
+  var maxEventsPerCollection: Integer = 10000
```

Comment:
Has to be a `var` if it's set via `eventStore.maxEventsPerCollection`, no? Also from the Java SDK.